crates/matrix-sdk-crypto/src/olm/group_sessions/outbound.rs

```diff
@@ -70,7 +71,7 @@ const ROTATION_MESSAGES: u64 = 100;
 pub(crate) enum ShareState {
     NotShared,
     SharedButChangedSenderKey,
-    Shared(u32),
+    Shared { message_index: u32, olm_wedging_index: SequenceNumber },
```

Comment:
This still isn't documented.

Reply:
Sorry, I misunderstood your original comment and thought that switching to named fields would avoid the need for documentation.  I've added docs, and while I was there, added docs for the rest of the enum too.